generator/generator.yml

```diff
@@ -101,3 +101,41 @@ modules:
       - 1.3.6.1.4.1.1718.3.2.2  # infeedTable
       - 1.3.6.1.4.1.1718.3.2.3  # outletTable
 
+# A10 Ax Load Balancers
+#
+# The MIB files are available for download through the load balancer GUI:
```

Comment:
We can't put these in this repo then. We need any random person to be able to run the generator with the included config.